src/components/post/Post.js

```diff
@@ -222,15 +221,15 @@ const Post = (props) => {
                       <>
                         <i
                           className="blue fas fa-chevron-up"
-                          onClick={() => unlike(postID)}
+                          onClick={onClickLikesHandler}
```

Comment:
Same question as Ian, does the postID need to be passed in? Otherwise looks great

Reply:
Am going to push a new request and this one can be deleted.